### DISCUSSION TITLE
Xournal on Raspbian with touch screen input

### DISCUSSION THREAD

**Discussion by nagylilla96** [xournalpp/xournalpp#2405] [Q&A] [answered]
Hi! I was wondering in Xournal was compatible with Raspbian with touch screen input. It doesn't seem to recognize any kind of touch screen devices, regardless of their Android version (I have tried a really old one (4.2) thinking that's the issue, but then I tried an 8.1 device as well and it didn't work). I installed xournal with `sudo apt install xournal`, so it's not xournal++, it's the old version of it, since I couldn't install xournal++ on the Raspberry Pi. Could you please give me your insights about this? Thanks!

**Comment by Technius** [comment, marked as answer]
I'm quite confused, are you talking about running [Xournal++ Mobile](https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile) on Android or running Xournal++ desktop (this repository) on Raspberry Pi? They are two different things (note to self: update the readme / website to mention that they are...).

CC @TheOneWithTheBraid

**Reply by nagylilla96** [reply]
I'm running Xournal on my Raspberry Pi and wish to use my phone as a touch screen input.

**Reply by Technius** [reply]
Ah, got it. You'll want to run something like [Weylus](https://github.com/H-M-H/Weylus) to enable your phone to be used as a touch screen device. This would work with Xournal at least.

Xournal++ currently hasn't been tested with ARM yet, so whether it actually works on Raspberry Pi is uncertain. If you're willing to test it out, you could compile it yourself and let us know if you run into any issues.

**Reply by nagylilla96** [reply]
Okay, I think I'll try Weylus then, and maybe I'll check out Xournal++ as well - installing it the recommended way did not work for me, but I might try building it. Thanks!
